test/index.test.js

```diff
@@ -1,41 +1,218 @@
 'use strict';
 
-const expect = require('chai').expect;
-const Hapi   = require('hapi');
+const Bluebird = require('bluebird');
+const expect   = require('chai').expect;
```

Comment:
if we don't put `expect` in `setup.js`, we usually put it as the first require (even above bluebird).